### PR TITLE
Implemented expanded row with additional info and handlers

### PR DESCRIPTION
**Pull request by hendrysiak**

### CODE REVIEW

File: src/shared/TableEditor/EditableRow.tsx [hunk 1]
```diff
@@ -32,141 +40,197 @@ const EditableRow = (props: Props): JSX.Element => {
...
+        <TableCell>
+          {props.letterReceived ? <Tooltip title="Wpłynęło pismo" aria-label="letter-received">
+            <IconButton><MailIcon/></IconButton>
+          </Tooltip> : <></>}
+          {props.dateOfLetter && props.dateOfLetter !== '' && props.dateOfLetter !== null ? <Tooltip title="Dodano datę pisma" aria-label="date-of-letter">
+            <IconButton><EventIcon/></IconButton>
+          </Tooltip> : <></>}
+          {props.comment && props.comment !== '' ? <Tooltip title="Dodano komentarz" aria-label="added-comment">
+            <IconButton><InsertCommentIcon/></IconButton>
+          </Tooltip> : <></>}
```
Comment (Tubka) [inline src/shared/TableEditor/EditableRow.tsx:188]: why you used ternary operation and return empty fragment JSX ? Maybe better to use logical && operation to return element or not return

Comment (hendrysiak) [reply src/shared/TableEditor/EditableRow.tsx:188]: This is the reason of using ternary:  
[https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx/](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx/)

Ofcourse - not at all cases need this method.

Comment (Tubka) [reply src/shared/TableEditor/EditableRow.tsx:188]: ok nice, but i still have a question, why return <></>, you not 'null' ?

**Comment by hendrysiak** [comment]
Merge after code review

**Pull request merged**